### ISSUE TITLE
how to set MediaStore to get image from specific folder?

### ISSUE THREAD

**Issue by hamaz562006** [kasem-sm/MediaStoreExample#1]

**Comment by kasem-sm** [comment]
You want to read whole Image or single image from a 'specific' folder?

**Comment by hamaz562006** [comment]
> You want to read whole Image or single image from a 'specific' folder?

If  android:requestLegacyExternalStorage="true" , I can get all the Images in a specific folder with use of MediaStore , but android:requestLegacyExternalStorage="true" not work.

According to Google's new policies, Is it still possible to use the tag android:requestLegacyExternalStorage="true" with api 30?

**Comment by kasem-sm** [comment]
> > You want to read whole Image or single image from a 'specific' folder?
> 
> If  android:requestLegacyExternalStorage="true" , I can get all the Images in a specific folder with use of MediaStore , but without tag android:requestLegacyExternalStorage="true" not work.
> 
> According to Google's new policies, Is it still possible to use the tag android:requestLegacyExternalStorage="true" with api 30?
> 
> 

No, you cant use that flag anymore on API 30, play console will not allow you to upload an aab or apk with that flag.
